### PR TITLE
Service Mesh Interface-2022-Annual Review

### PR DESCRIPTION
**Pull request by keithmattix**
Adding 2022 SMI review per https://github.com/cncf/toc/blob/main/process/sandbox-annual-review.md.

Signed-off-by: Keith Mattix II <keithmattix2@gmail.com>

### CODE REVIEW

File: reviews/2022-openservicemesh-annual.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+- *Include a link to your project’s devstats page. We will be looking for signs of consistent or increasing contribution activity. Please feel free to add commentary to add colour to the numbers and graphs we will see on devstats.*
+  - Year in review for the main [SMI devstats dashboard](https://smi.devstats.cncf.io/d/8/dashboards?orgId=1&from=now-365d&to=now-1h&refresh=1d)
+    - This shows a sustained, consistent level of contributions to the spec with changes based on discussion between maintainers and the community (with an expected dropoff around KubeCon events and the holidays). While devstats can show releases, repo activity is not the only metric for a specification. It’s useful to consider nuance in the content of the updates (specifically, how it’s been clarified due to direct input from implementers.)
+  - Participation from [various organizations](https://smi.devstats.cncf.io/d/5/companies-table?orgId=1)
+
+- *How many maintainers do you have, and which organisations are they from? (Feel free to link to an existing MAINTAINERS file if appropriate.)*
+  - SMI currently has [12 core maintainers](https://github.com/servicemeshinterface/smi-spec/blob/main/CODEOWNERS)
```
Comment (leecalcote) [inline reviews/2022-openservicemesh-annual.md:34]: Project needs to evaluate and invoke inactive maintainers governance.

Comment (bridgetkromhout) [reply reviews/2022-openservicemesh-annual.md:34]: Sure thing, but that hasn't happened yet, so this report is stating where we're at currently! Thanks.

**Comment by dims** [comment]
@dzolotusky thanks for picking this up to be the TOC shepherd for SMI (for the 6/21 TOC meeting)

**Comment by dzolotusky** [comment]
@keithmattix The TOC met yesterday and talked through the annual reviews.
For SMI, we are happy with the content of your annual review and the progress that the project has made. The two big things that we'd like to engage more with the project on are:

1. SMI's status for incubation. In your annual review, you wrote "While some details appear to be focused on suitable metrics for source code, not necessarily specifications, we think we can meet the criteria for incubating projects.". We're like to talk a bit more about how close to applying the project is, and to discuss more about how some of the metrics are more suitable for source code. Maybe we can work with you to improve the incubation guidelines to make them better applicable to both source code and specs?
2. Your response to "How can the CNCF help you achieve your upcoming goals?" only spoke about things that have already happened to help SMI. Is there more that you would like to have the from the CNCF or the TOC to help the project?

If you'd like more details from that meeting, the slides are here: https://docs.google.com/presentation/d/1ARxZNb5orglbugP8CWn7ssqH-R_xidP6UMCaxZnN0Yo

**Pull request merged**